galite-core/src/main/kotlin/org/kopi/galite/visual/ui/vaadin/form/DTextField.kt

```diff
@@ -74,6 +73,9 @@ open class DTextField(
 
     field.inputField.addTextValueChangeListener {
       if (it.isFromClient) {
+        if (getModel().getDataType() == LocalDate::class){
+          getModel().block!!.activeField = getModel()
+        }
```

Comment:
We should generalise this. Since the problem is with field focus, do this instead : 
        if (!getModel().hasFocus()) {
          getModel().block!!.gotoField(getModel())
        }

Also, should we add the same check to DGridTextEditorField ?

Reply:
Using this treatment inside the old one returns the same results. As for whether we should add the same check to DGridTextEditorField, we don’t need that because grid fields are not editable in Galite. However, if the behavior changes, it may be necessary.